### PR TITLE
fix bug on model definition part of introduction

### PR DESCRIPTION
**Pull request by schnell3526**
This PR is related to issue https://github.com/pytorch/tutorials/issues/1526

A relu layer was inserted just before the cross entropy error was calculated, and it seemed that the loss was not calculated correctly, so I removed the relu layer.
Accordingly, the structure of the NeuralNetwork class in buildmodel_tutorial.py was also changed.

If you have time, I hope you will take a look.

### CODE REVIEW

**Comment by suraj813** [comment]
Good catch! Thank you @sehyun03 and @schnell3526  for fixing this

**Review by suraj813 — APPROVED** [review]
